tests/integration/cluster_test.go

```diff
@@ -74,8 +72,7 @@ func TestClusterOf3UsingDiscovery(t *testing.T) { testClusterUsingDiscovery(t, 3
 
 func testClusterUsingDiscovery(t *testing.T, size int) {
 	integration.BeforeTest(t)
-	dc := integration.NewCluster(t, 1)
-	dc.Launch(t)
+	dc := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1, UseIP: true})
 	defer dc.Terminate(t)
 	// init discovery token space
 	dcc := integration.MustNewHTTPClient(t, dc.URLs(), nil)
```

Comment:
What's our strategy with discovery (testing). 

Migrate to e2e test with loading 3.5 binary as a discovery server ?

Reply:
SG